exercises/palindrome/src/example.c

```diff
@@ -0,0 +1,85 @@
+#include "palindrome.h"
+#include "stdio.h"
+#include "assert.h"
+#include "string.h"
+#include "stdlib.h"
+
+/* Return 0 if the product of a and b is a palindrome, 1 otherwise */
+int isPalindrome(int a, int b)
```

Comment:
Instead of using an `int` for your result, how about using `stdbool.h` so that you can use `true` and `false`?